test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -235,12 +237,14 @@ TEST_F(HttpFilterTest, TimeoutError) {
   response.status = Filters::Common::ExtAuthz::CheckStatus::Error;
   response.error_kind = Filters::Common::ExtAuthz::ErrorKind::Timedout;
   request_callbacks_->onComplete(std::make_unique<Filters::Common::ExtAuthz::Response>(response));
-  EXPECT_EQ(
-      1U,
-      filter_callbacks_.clusterInfo()->statsScope().counterFromString("ext_authz.error").value());
-  EXPECT_EQ(
-      1U,
-      filter_callbacks_.clusterInfo()->statsScope().counterFromString("ext_authz.timeout").value());
+  EXPECT_EQ(1U, filter_callbacks_.clusterInfo()
+                    ->statsScope()
+                    .counterFromString("ext_authz.slow_service.error")
```

Comment:
Suggest an explicit test for this rather than bolting onto timeout.

Reply:
Done.